topics/telegram/index.md

```diff
@@ -0,0 +1,12 @@
+---
+aliases: telegram-messenger, telegram-client, telegram-bot, telegram-api
```

Comment:
`telegram-messenger` isn't in use on any public repositories currently, see https://github.com/topics/telegram-messenger. Could you remove it?

Reply:
Done.